### PR TITLE
Update RHEL support for GCE

### PR DESCRIPTION
**Pull request by trevor-vaughan**
Added /rhel/ matches to all appropriate areas to support GCE naming
conventions

This is required by the GCE fixes in beaker-google

### CODE REVIEW

**Comment by trevor-vaughan** [comment]
@kevpl Ok, why did this fail? It certainly works in production.

**Comment by mchllweeks** [comment]
retest this please

File: .rspec [hunk 1]
```diff
@@ -1,3 +1,4 @@
 --format documentation
 --color
 --tty
+--fail-fast
```
Comment (tvpartytonight) [inline .rspec]: @trevor-vaughan can you let us know why you are adding this flag? It seems like if you want faster feedback while iterating on spec tests, you should run only the tests you care about by passing in a file name. When this runs in CI, we will want to see all test errors, not just the first one encountered.

Comment (trevor-vaughan) [reply .rspec]: Sorry about that. I'll pull that back out.

**Comment by trevor-vaughan** [comment]
@tvpartytonight Why is Jenkins failing?

**Comment by ferglor** [comment]
@puppetlabs-jenkins retest this please

**Comment by ferglor** [comment]
@trevor-vaughan can you resolve the conflicts in this PR please?

**Comment by trevor-vaughan** [comment]
@ferglor Not sure when I'll have time. The PR allows commits by maintainers if you want to just rebase it against my branch.

**Comment by trevor-vaughan** [comment]
Had a minute and rebase went quickly

**Comment by trevor-vaughan** [comment]
Had a minute and proved to be easier than expected

**Comment by trevor-vaughan** [comment]
image

**Comment by kevpl** [comment]
blocked on conversation in https://github.com/puppetlabs/beaker-puppet/pull/20#issuecomment-340506667

**Pull request closed without merging**